clients/drcachesim/analyzer.h

```diff
@@ -167,6 +168,7 @@ template <typename RecordType, typename ReaderType> class analyzer_tmpl_t {
         }
 
         uint64_t cur_interval_index;
+        // Cumulative instr count when the current interval started (non-inclusive).
```

Comment:
How about "Cumulative count of instructions before the start of the current interval."?  I find "non-inclusive" describing a specific count (rather than a range) to be a bit confusing.

Reply:
I agree. Done.